modules/secure-serverless-harness/variables.tf

```
@@ -19,12 +19,13 @@ variable "billing_account" {
   type        = string
 }
 
-variable "serverless_type" {
-  description = "The type of resource to be used. It supports only CLOUD_RUN or CLOUD_FUNCTION"
+variable "base_serverless_api" {
+  description = "This variable will enable Cloud Function or Cloud Run API, based in which resources will be used in the harness. Cloud Run API will be used for the terraform-google-cloud-run repository while Cloud Function API will be used in the terraform-google-cloud-functions repository. It supports only run.googleapis.com or cloudfunctions.googleapis.com"
```

The Cloud Function api is not being enabled by harness bases on this variable, is being enabled by the serverless_extra_apis. The description is not matching the behavior.
```suggestion
  description = "This variable will enable Cloud Function or Cloud Run API specific resources. Cloud Run API will be used for the terraform-google-cloud-run repository while Cloud Function API will be used in the terraform-google-cloud-functions repository. It supports only run.googleapis.com or cloudfunctions.googleapis.com"
```